src/env.cc.in

```diff
@@ -146,11 +160,12 @@ const std::vector<std::string> Env::cyclus_path() {
   std::cout << __FILE__ << ":" << __LINE__ << ": "#X" = " << X << "\n"
 
 std::string Env::FindModule(std::string path) {
+  boost::system::error_code errc;
   std::vector<std::string> strs = cyclus_path();
 
   for (int i = 0; i < strs.size(); ++i) {
     fs::path full = fs::path(strs[i]) / path;
-    if (fs::exists(full)) {
+    if (fs::exists(full, errc)) {
```

Comment:
Is this change due to an API Change in boost?  It doesn't appear that the `errc` is ever used.